docs/source/install.rst

```diff
@@ -13,51 +13,53 @@ Installation
 Installing with pip
 -------------------
 
-It's possible to install ``libsemigroups_pybind11`` using ``pip`` via one of:
+It's possible to install ``libsemigroups_pybind11`` using ``pip`` something
+like the following (depending on your system and setup):
 
 ::
 
     pip install libsemigroups_pybind11
     pip3 install libsemigroups_pybind11
     python -m pip install libsemigroups_pybind11
 
-Installing with conda
----------------------
 
-This installation method assumes that you have anaconda or miniconda installed.
-See the `getting started`_ and `miniconda download page`_ on the conda_ website.
-
-.. _conda: https://conda.io/
-
-.. _getting started: http://bit.ly/33B0Vfs
-
-.. _miniconda download page: https://conda.io/miniconda.html
-
-It might be a good idea to create and activate a conda environment to contain
-the installation of the ``libsemigroups_pybind11``:
-
-::
-
-    conda create --name libsemigroups
-    conda activate libsemigroups
-
-Install ``libsemigroups_pybind11``:
-
-::
-
-    conda install -c conda-forge libsemigroups_pybind11
-
-.. warning::
-
-    At present this does not work for Macs with M1 processors.
+..  Installing with conda
+    ---------------------
+    
+    This installation method assumes that you have anaconda or miniconda installed.
+    See the `getting started`_ and `miniconda download page`_ on the conda_ website.
+    
+    .. _conda: https://conda.io/
+    
+    .. _getting started: http://bit.ly/33B0Vfs
+    
+    .. _miniconda download page: https://conda.io/miniconda.html
+    
+    It might be a good idea to create and activate a conda environment to contain
+    the installation of the ``libsemigroups_pybind11``:
+    
+    ::
+    
+        conda create --name libsemigroups
+        conda activate libsemigroups
+    
+    Install ``libsemigroups_pybind11``:
+    
+    ::
+    
+        conda install -c conda-forge libsemigroups_pybind11
+    
+    .. warning::
+    
+        At present this does not work for Macs with M1 processors.
 
 From the sources
 ----------------
 
 Before installing ``libsemigroups_pybind11`` from its sources you should first
 perform a system install of the C++ library ``libsemigroups``. For information
 about how to install ``libsemigroups`` see `the installation guide
-<https://libsemigroups.readthedocs.io/en/latest/install.html>`_.
+<https://libsemigroups.github.io/libsemigroups/md_install.html>`_.
```

Comment:
This might be clearer as: "For information about how to do this, see [the libsemigroups installation guide](https://libsemigroups.github.io/libsemigroups/md_install.html)"